install.js

```diff
@@ -0,0 +1,63 @@
+const { existsSync, mkdirSync, createWriteStream } = require("fs");
+const axios = require("axios");
+
+const pkg = require("./package.json");
+
+const BUNDLETOOL_VERSION = pkg.bundletool.version;
+
+const { DEFAULT_BASE_DIRECTORY, BUNDLETOOL_FILE_PATH } = require("./bin");
+const BUNDLETOOL_URL = `https://github.com/google/bundletool/releases/download/${BUNDLETOOL_VERSION}/bundletool-all-${BUNDLETOOL_VERSION}.jar`;
+
+const log = (message) => {
+  const logLevel = process.env.npm_config_loglevel;
+  const logLevelDisplay = ["silent", "error", "warn"].indexOf(logLevel) > -1;
```

Comment:
Any specific reason for `.indexOf` instead of `.includes`?

Reply:
Nope. This code was borrowed from the install script in `pupeteer` and I just didn't change it. I'm guessing they were using the `indexOf` method for compatibility with older versions of Node.